hr_timesheet_sheet_validators/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-17 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'HR Timesheet Sheet Validators',
+    'version': '10.0.1.0.0',
+    'author': 'Eficent, Serpent Consulting Services Pvt. Ltd.',
```

Comment:
As I said to you in a module yesterday and I am seeing that you doing some more... if you are not adding capabilities, migrations doesn't grant authorship.